frontend/packages/git-service/src/utils/pac-strategy-detector.ts

```diff
@@ -0,0 +1,60 @@
+import { BaseService } from '../services/base-service';
+
+export const detectPacFiles = async (gitService: BaseService) => {
+  let pacFiles = [];
+  const isTektonFolderPresent = await gitService.isTektonFolderPresent();
```

Comment:
Is this call really required? We could check if `.tekton` does exist in the files list before calling detectPacFiles?!